atmo/jobs/models.py

```diff
@@ -76,21 +78,11 @@ class SparkJob(EMRReleaseModel, CreatedByModel):
         default=True,
         help_text="Whether the job should run or not."
     )
-    last_run_date = models.DateTimeField(
-        blank=True,
-        null=True,
-        help_text="Date/time that the job was last started, null if never."
-    )
-    current_run_jobflow_id = models.CharField(
+    last_jobflow_id = models.CharField(
```

Comment:
Right. I started w/o it but there was one case where it was nice to not have to do a query to get it.